config/environments/production.rb

```diff
@@ -9,57 +9,68 @@
   }
 
   config.cache_store = :dalli_store,
-  (ENV["MEMCACHIER_SERVERS"] || "").split(","),
-  {:username => ENV["MEMCACHIER_USERNAME"],
-   :password => ENV["MEMCACHIER_PASSWORD"],
-   :failover => true,
-   :socket_timeout => 1.5,
-   :socket_failure_delay => 0.2
+    (ENV["MEMCACHIER_SERVERS"] || "").split(","),
+    {:username => ENV["MEMCACHIER_USERNAME"],
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.